blockchain/blockchain.go

```diff
@@ -266,6 +263,9 @@ func NewBlockChain(db database.DBManager, cacheConfig *CacheConfig, chainConfig
 		return nil, err
 	}
 
+	// Initialize DeriveSha implementation
```

Comment:
@hqjang-pepper 
What is the reason of this change? It seems the value is not changed.